index.html

```diff
@@ -205,5 +205,31 @@ <h2 class="about-me-topper">About me</h2>
         </div>
 
     </section>
-    </body>
-    </html>
\ No newline at end of file
+
+    <section class="form">
+        <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        <form class="contact_form" action="https://formspree.io/f/mayzodlp" method="post">
+            <ul class="form-inputs">
+                <li><input type="text" placeholder="Full name" required maxlength="30" id="name" name="user_name"></li>
+                <li><input type="email" placeholder="Email address" id="email" name="user_email"></li>
```

Comment:
- [ ] As mentioned in the project requirements please add the required attribute to email input also, currently it is possible to submit the form without an email, please fix it